firefox-ios/firefox-ios-tests/Tests/XCUITests/OnboardingTests.swift

```diff
@@ -74,6 +80,81 @@ class OnboardingTests: BaseTestCase {
         app.buttons["\(rootA11yId)PrimaryButton"].tap()
         let topSites = app.collectionViews.cells[AccessibilityIdentifiers.FirefoxHomepage.TopSites.itemCell]
         mozWaitForElementToExist(topSites)
+        switchThemeToDarkOrLight(theme: "Dark")
```

Comment:
I am a bit concerned about this test checking both themes.. it is long, wondering if it would be better to have two different tests instead

Reply:
I've separated the tests for Light and Dark mode